README.md

```diff
@@ -37,26 +37,26 @@ Building Halide
 
 #### TL;DR
 
-Have llvm-6.0 or greater installed and run `make` in the root
+Have llvm-7.0 or greater installed and run `make` in the root
 directory of the repository (where this README is).
 
 #### Acquiring LLVM
 
-Building Halide requires at least LLVM 6.0, along with the matching
+Building Halide requires at least LLVM 7.0, along with the matching
 version of Clang. `llvm-config` and `clang` must be somewhere in the
-path. If your OS does not have packages for llvm-6.0, you can find
+path. If your OS does not have packages for llvm-7.0, you can find
 binaries for it at http://llvm.org/releases/download.html. Download an
 appropriate package and then either install it, or at least put the
 `bin` subdirectory in your path. (This works well on OS X and Ubuntu.)
 
 If you want to build it yourself, first check it out from subversion:
 
-    % svn co https://llvm.org/svn/llvm-project/llvm/branches/release_60 llvm6.0
-    % svn co https://llvm.org/svn/llvm-project/cfe/branches/release_60 llvm6.0/tools/clang
+    % svn co https://llvm.org/svn/llvm-project/llvm/branches/release_70 llvm7.0
```

Comment:
Should probably update the instructions to 80

Reply:
dangit, thought I caught all of these :-(

Reply:
I has always been the min version because we've only ever updated it when we drop a version :)